persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraKeyspaceConfig.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.event.LoggingAdapter
+import com.typesafe.config.Config
+
+private[lagom] object CassandraKeyspaceConfig {
+
+  def validateKeyspace(namespace: String)(implicit config: Config, log: LoggingAdapter): Unit =
+    validateKeyspace(namespace, s"$namespace.defaults")
```

Comment:
I think this method provides little value. It's only valid in 2/3 of the cases and it confused me a bit when I found the other 1/3 of the cases. The final `path` being validated via this execution path is the concat of up to three pieces: `s"$namespace.defaults.keyspace"`.
Also, I don't think using implicit parameters here helps. When calling the method I have to create two implicit value (two extra lines of code) or pass the values explicitly anyway (which I'd prefer).

I do like that the logger is passed in so that it's the caller's logging scope.